.github/workflows/deploy.yml

```
@@ -612,11 +612,18 @@ jobs:
       - name: Download all workflow run artifacts
         uses: actions/download-artifact@95815c38cf2ff2164869cbab79da8d1f422bc89e
 
+      - name: Get last production release
+        id: get_last_prod_version
+        run: echo "LAST_PROD_VERSION=$(gh release list --repo ${{ github.repository }} --exclude-drafts --exclude-pre-releases --limit 1 --json tagName -q '.[0].tagName')" >> "$GITHUB_OUTPUT"
```

For clarity (I always prefer `--long-form-flags` in CI)

```suggestion
        run: echo "LAST_PROD_VERSION=$(gh release list --repo ${{ github.repository }} --exclude-drafts --exclude-pre-releases --limit 1 --json tagName --jq '.[0].tagName')" >> "$GITHUB_OUTPUT"
```

---

Oh nice I agree. I didn't know that `-q` is actually `-jq`